assets/js/script.js

```diff
@@ -1,87 +1,101 @@
 // NewsAPI
 
-let strNewsAPIKey = "0e966fb836610aa7a1a213a0b9d61c6b";
-let strCountryCode = "au";
+let newsAPIKey = "0e966fb836610aa7a1a213a0b9d61c6b";
+let countryCode = "au";
 let newsQueryURL = `https://gnews.io/api/v4/top-headlines?country=${countryCode}&max=5&token=${newsAPIKey}`;
```

Comment:
- The newsAPIKey and newsQueryURL variables need to be moved into the getNews function which I forgot to do earlier.
- The ${countryCode} variable inside the newsQueryURL string needs to be changed to 'catchCountry' to match the variable name you're passing from your function.
- The countryCode variable declaration can be deleted entirely.

Reply:
Have made the appropriate changes, please review and comment/approve as necessary.